content/docs/09.administrator-guide/02.deployment/04.restack.md

```diff
@@ -1,56 +1,190 @@
 
 To Sign up for a Restack account, visit [www.restack.io/signup](https://www.restack.io/signup). You can sign up with your corporate email address or your GitHub profile. You do not need a credit card to sign up.
 ![restack-signup](/docs/administrator-guide/deployment/restack/restack-sign-up.png)
 If you already have an account, login to Restack at [www.restack.io/login](https://www.restack.io/login).
 
-## Add AWS credentials with AdministratorAccess
+## Connect your AWS account
 
-To deploy Kestra in your own AWS infrastructure with Restack, you will need to add your credentials as the next step.
+Log into Restack Console and navigate to Settings > Cloud > Connect Cloud
+![](https://cdn.sanity.io/images/ev3amoz3/production/4c3c9385fd468fd49fcade0b405f089e14303bba-4320x2280.png)
 
-Make sure that this account has *AdministratorAccess*. This is how Restack can ensure an end-to-end cluster creation and cluster management process.
+Create an IAM Role by following these steps:
 
-1. Navigate to **Clusters** in the left-hand navigation menu.
-2. Select the **Credentials** tab.
-3. Click **Add credential**.
-    ![add-credentials](/docs/administrator-guide/deployment/restack/restack-add-credentials.png)
-4. Give a suitable title to your credentials for managing them later.
-5. Enter your [**AWS Access Key ID** and **AWS Secret Access key**.](https://docs.aws.amazon.com/accounts/latest/reference/root-user-access-key.html)
-6. Click **Add credential**.
-    ![add-aws-credentials](/docs/administrator-guide/deployment/restack/restack-add-AWS-creds.png)
+1. Log into your AWS Management Console as a user with administrator privileges and go to the IAM console.
 
->[How to get your AWS Access key ID and AWS Secret Access Key](https://docs.aws.amazon.com/accounts/latest/reference/root-user-access-key.html)
+2. Click the Roles tab and click on Create Role in the IAM sidebar.
 
-## One-click cluster creation with Restack
+ ![](https://cdn.sanity.io/images/ev3amoz3/production/7d1ba16ba5610e8947ca20d8746ee8424a9d4621-2000x1062.png)
+
+3. In Select type of trusted entity, click the Custom trust policy
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/666cb5015b3c5a29e6eef6216bdd97ef67297dc8-1770x1110.png)
+
+4. Copy and Paste into the custom policy field the following. You also can copy it directly from the Restack user interface modal.
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": [
+                    "arn:aws:iam::223746943638:user/connect",
+                    "arn:aws:iam::223746943638:role/connect"
+                ]
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "sts:ExternalId": "restack"
+                }
+            }
+        }
+    ]
+  }
+```
+
+5. Click the Next button and Add AdministratorAccess Permissions.
```

Comment:
```suggestion
5. Click the "Next" button and under the "Add permissions" section, add the `AdministratorAccess` policy.
```

out of curiosity, why Admin access is required?

Reply:
Admin access is the default one for companies that want a fully managed solution (the majority). Thanks to that we can provision and manage all services needed to run the products without the customer updating permissions every time we need to manage a new one. 
For enteprise customers that want more restrictive permissions, they should contact us and we will offer a more restricted version that they will need to update. We should probably add that to the documentation. I will send a commit with my suggested changes.